site/docs/install-os-x.md

```diff
@@ -101,24 +101,12 @@ Install Homebrew (a one-time step):
 https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 ```
 
-### Step 2: Install the Bazel Homebrew package
-
-Please note that there are two different Bazel packages ("formulae"):
-
-* `bazelbuild/tap/bazel` is maintained by the Bazel team, and installs the
-  official Bazel binary package (see above).
-* The default [homebrew/core formula](https://formulae.brew.sh/formula/bazel)
-  is maintained by the community, and compiles Bazel from source or installs a
-  binary pre-compiled by the Homebrew project.
-
-If you previously installed Bazel from homebrew/core, you can switch to the
-`bazelbuild` tap by uninstalling the existing package: `brew uninstall bazel`.
+### Step 2: Install Bazel via Homebrew
 
 Install the `bazelbuild/tap` Bazel package via Homebrew as follows:
```

Comment:
The instructions no longer install the `bazelbuild/tap` package, so this should probably just read:

"Install the Bazel package via Homebrew as follows:"

Reply:
Yeah, this actually also reminds me another pull request needs to be filed to the tap (so that the tap users can be migrated to use the core one)

Reply:
I also filed a separate PR for the tap regarding the formula installation in the future.